src/router.ts

```diff
@@ -22,9 +22,11 @@ export class Router {
       await this.updater.handlePush();
     } else if (eventName === 'workflow_run') {
       await this.updater.handleWorkflowRun();
+    } else if (eventName === 'schedule') {
+      await this.updater.handleSchedule();
     } else {
       throw new Error(
-        `Unknown event type '${eventName}', only 'push', 'pull_request' and 'workflow_run' are supported.`,
+        `Unknown event type '${eventName}', only 'push', 'pull_request', 'workflow_run', and 'schedule' are supported.`,
```

Comment:
A thought: can `workflow_run` trigger `schedule` events? If so - do we need to update the workflow run handling?